gammapy/modeling/models/cube.py

```diff
@@ -648,6 +648,7 @@ def __init__(
         spectral_model=None,
         spatial_model=None,
         covariance_data=None,
+        name=None,
```

Comment:
These 2 fields should be described into the doctring, with their default values.